src/notation/view/notationcontextmenumodel.cpp

```diff
@@ -80,7 +80,22 @@ MenuItemList NotationContextMenuModel::makeDefaultCopyPasteItems()
 
 MenuItemList NotationContextMenuModel::makeMeasureItems()
 {
-    MenuItemList items = makeElementItems();
+    MenuItemList items = {
+        makeMenuItem("notation-cut"),
+        makeMenuItem("notation-copy"),
+        makeMenuItem("notation-paste"),
+        makeMenuItem("notation-swap"),
+    };
+
+    items << makeSeparator();
+
+    MenuItem* clearItem = makeMenuItem("notation-delete");
+    clearItem->setTitle("Clear measures");
```

Comment:
You missed translations here. Should be:

clearItem->setTitle(qtrc("notation", "Clear measures"))

The same for deleteItem

Reply:
Thanks for pointing it out. I already corrected it.